test/test_retrieve_and_rank_v1.py

```diff
@@ -0,0 +1,225 @@
+import os
+import responses
+import watson_developer_cloud
+
+@responses.activate
+def test_success():
+    
+# Ranker endpoints
+    retrieve_and_rank = watson_developer_cloud.RetrieveAndRankV1(username="username",
+                                                                 password="password")
+
+    listrank_url = 'https://gateway.watsonplatform.net/retrieve-and-rank/api/v1/rankers'
+    listrank_response = '{"rankers":[{"ranker_id":"3b140ax14-rank-10383","url":"https://gateway.watsonplatform.net/retrieve-and-rank' \
+                    '/api/v1/rankers/3b140ax14-rank-10383","name":"pythonRank","created":"2016-07-15T19:11:27.801Z"}]}'
+                    
+    responses.add(responses.GET, listrank_url,
+              body=listrank_response, status=200,
+              content_type='application/json')
+
+    retrieve_and_rank.list_rankers()
```

Comment:
if you save the `list_rankers` result into a variable you can then check that the number of rankers match the static response.

```python
ranker_list = retrieve_and_rank.list_rankers()
assert len(ranker_list['rankers']) == 1
```